src/App/ExternalServices/HDXInterface.php

```diff
@@ -21,12 +21,14 @@ class HDXInterface
 {
     protected $ckanClient;
     protected $userAPIKey;
+    protected $hdx_user_id;
```

Comment:
I know I approved the spec, and this matches it, so feel free to ignore this and add a tech debt task for me... but I really think this should be called hdx_maintainer. There's an id in HDX for users that they return in their API (different than maintainer) and it opens things up for more confusion if we use $hdx_user_id for what is actually the maintainer.

Reply:
ok that's a little annoying but it's just a find and replace so I can do it today.